transformers/shared/constants/method.go

```diff
@@ -83,19 +84,19 @@ func VatABI() string  { return getContractABI("MCD_VAT") }
 func VowABI() string  { return getContractABI("MCD_VOW") }
 
 func auctionFileMethod() string { return getSolidityFunctionSignature(FlipABI(), "file") }
-func biteMethod() string        { return getSolidityFunctionSignature(CatABI(), "Bite") }
+func biteMethod() string        { return getSolidityFunctionSignature(Cat100ABI(), "Bite") }
```

Comment:
I'm kind of wondering if we want to use `Cat110ABI` instead here, just to rely on the most recent contract? They should be the same, so it probably doesn't matter too much 🤔

Reply:
Why wouldn't we use the latest? Strokes me as odd to use the old one, but I am a Ether-dumb-dumb.

Reply:
I think using the latest one makes sense - for many of the cat methods, the ABI didn't change, so it doesn't matter too much. But might as well use the most recent one. I'll push up a commit shortly!